store/store.go

```diff
@@ -11,6 +12,10 @@ func NewCommitMultiStore(db dbm.DB) types.CommitMultiStore {
 	return rootmulti.NewStore(db)
 }
 
+func NewCommitKVStoreCacheManager() types.MultiStorePersistentCache {
+	return cache.NewCommitKVStoreCacheManager(cache.DefaultCommitKVStoreCacheSize)
```

Comment:
Should this be a end-user-configurable paramater (in some `.toml` file)?

Reply:
Added `--inter-block-cache` flag to the server package with `true` by default.